src/app.py

```diff
@@ -42,34 +60,40 @@ def index():
     @write_required
     def add_item():
         name = request.form['name']
+        app.logger.debug(f"User {current_user.id} added item '{name}'")
```

Comment:
Given the message, this belongs after it was added successfully, and also it would be better to log the id rather than the name (as you do elsewhere).

In your other functions, again the logging should be after the action was completed successfully as the messages claim that the action completed.